<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="cs8MCPDisplay" name="widget" native="true"/>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>CheckBox</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Mode</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbMode">
        <item>
         <property name="text">
          <string>Normal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Inversed</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Blinking</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Inv Blink</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Row</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spRow">
        <property name="maximum">
         <number>13</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Column</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spColumn">
        <property name="maximum">
         <number>39</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Text</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leText">
        <property name="maxLength">
         <number>40</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbPut">
        <property name="text">
         <string>Put</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbCLS">
        <property name="text">
         <string>CLS</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>729</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>cs8MCPDisplay</class>
   <extends>QWidget</extends>
   <header>../lib/src/cs8mcpdisplay.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
